automated-comments/go/two-fer/plus_used_for_concatenation.md

```diff
@@ -0,0 +1,3 @@
+In this case using `+` is almost certainly the right choice, as it's shorter in this case,
```

Comment:
Here we have "in this case" twice in the same phrase. Maybe use "here" for one of them?

Reply:
Changed it to
`In this case using `+` is almost certainly the right choice, as it's shorter,
and often faster for simple concatenations.`

Does that sound good?

Reply:
Not sure if the `,` before the `and` should stay. In German it would have to be removed.

Reply:
Grammatically it's fine with or without. I think I prefer it being there as it adds clarity that the shortness and fastness are two separate concepts worth taking a breath in between.